src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -91,6 +91,7 @@ const AccordionPanel = forwardRef(
           role="tab"
           aria-selected={active}
           aria-expanded={active}
+          plain={theme.button.default}
```

Comment:
Since plain is boolean, I think this should be `!!theme.button.default`

Reply:
Right. 
Though, in case `theme.button.default` is undefined, `!!theme.button.default` will be `false`, and having plain false wouldn't be the desired behavior, so I set it to 
`plain={theme.button.default ? true : undefined}`

Reply:
waiting on https://www.githubstatus.com/incidents/3q541nk49qmw to push the fix